items.js

```diff
@@ -250,9 +250,13 @@ class Fireball {
                 } else if (entity instanceof Tube || entity instanceof SideTube || entity instanceof Block || 
                     (entity instanceof Brick && (that.BB.collide(entity.leftBB) || that.BB.collide(entity.rightBB)))) {
                     that.explode = true;
-                } else if ((entity instanceof Goomba || entity instanceof Koopa) && !entity.dead) {
+                } else if ((entity instanceof Goomba || entity instanceof Koopa || entity instanceof PirahnaPlant) && !entity.dead) {
                     entity.dead = true;
                     that.explode = true;
+
+                    if (entity instanceof PirahnaPlant && that.BB.collide(entity.tube.BB)) {
```

Comment:
I'm not sure this will correctly protect the pirahnaplant, won't it make it invincable in all cases except when its fully out?

can't actually check right now however.